builtin_string.go

```diff
@@ -157,24 +157,38 @@ func (r *Runtime) string_raw(call FunctionCall) Value {
 	}
 }
 
+func (r *Runtime) stringproto_at(call FunctionCall) Value {
+	r.checkObjectCoercible(call.This)
+	s := call.This.toString()
+	pos := int(call.Argument(0).ToInteger())
```

Comment:
This will not work correctly on 32-bit architectures. Please implement as suggested.

Reply:
done